src/commands/moderation.rs

```diff
@@ -216,7 +220,7 @@ pub async fn kick(
 )]
 pub async fn timeout(
     ctx: Context<'_>,
-    #[description = "The member to timeout"] mut member: Member,
+    #[description = "The member to (un)timeout"] mut member: Member,
```

Comment:
This should be responsible of only timing people out, not un-timing out them (as there is another command to do so already)